app/models/user.rb

```diff
@@ -0,0 +1,14 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable and :omniauthable
+  has_many :mountains,
+    class_name: 'Mountain',
```

Comment:
You should not need to specify a `class_name` here since it goes with convention: `has_many :mountains` will look for a class of `Mountain` by default.